secret-store/src/types/error.rs

```diff
@@ -191,3 +191,12 @@ impl From<net::AddrParseError> for Error {
 		Error::Internal(err.to_string())
 	}
 }
+
+impl From<secp256k1::Error> for Error {
+	fn from(e: secp256k1::Error) -> Self {
+		match e {
+			secp256k1::Error::InvalidSecretKey => Error::EthKey("Invalid SecretKey".into()),
+			_ => Error::EthKey("Unknown crypto error".into())
```

Comment:
maybe use a display impl of `secp256k1::Error` here instead of `Unknown crypto error`?